### DISCUSSION TITLE
求助！如何加点？

### DISCUSSION THREAD

**Discussion by yang1cong2** [share121/inter-knot#749] [其他]
![微信图片_20240810160557](https://github.com/user-attachments/assets/ad7a6a03-68d6-43a5-8867-63fde5a4c0c6)

**Comment by feco-y** [comment]
哈哈我也有这张图

**Comment by piaomiaoxing** [comment]
把点一半加耐力一半加力量,留一个点给智力。 技能:格挡、暴击、冲锋、旋风舞、斩首、破甲。 法系技能点个圣光就行了。

**Comment by Nirvana341** [comment]
全点近战，要什么魔法，直接上去干就完啦
